src/main/java/edu/harvard/iq/dataverse/dataverse/messages/LazyDataverseTextMessage.java

```diff
@@ -0,0 +1,77 @@
+package edu.harvard.iq.dataverse.dataverse.messages;
+
+import com.google.common.collect.Lists;
+import edu.harvard.iq.dataverse.dataverse.messages.dto.DataverseLocalizedMessageDto;
+import edu.harvard.iq.dataverse.dataverse.messages.dto.DataverseMessagesMapper;
+import edu.harvard.iq.dataverse.dataverse.messages.dto.DataverseTextMessageDto;
+import org.primefaces.model.LazyDataModel;
+import org.primefaces.model.SortOrder;
+
+import javax.ejb.EJB;
+import javax.ejb.Stateful;
+import javax.inject.Inject;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+@Stateful
+public class LazyDataverseTextMessage extends LazyDataModel<DataverseTextMessageDto> {
+
+    @EJB
+    private DataverseTextMessageServiceBean dataverseTextMessageService;
+
+    @Inject
+    private DataverseMessagesMapper mapper;
+
+    private Long dataverseId;
+    private List<DataverseTextMessageDto> dataverseTextMessageDtos;
```

Comment:
Można wywalić bo i tak zwracamy dane

Reply:
W sumie to zrobilem to na zapas aby moc uzyc w getRowData() jakby sie koncepcja zmienila i np. ktos chcialby kliknac.